docs/support_playbook.md

```diff
@@ -116,6 +123,14 @@ If the current application status is `offer` use ChangeOffer service.
 
 This is possible via the support UI.
 
+### Adding offer conditions when application choice status is not 'offer'
```

Comment:
Im not too sure around adding this, doesn't feel too common 🤔 I'd suggest we hold off for now